### PR TITLE
Improve some config setting descriptions

### PR DESCRIPTION
**Pull request by johnnovak**
What the title says 😎

### CODE REVIEW

**Comment by johnnovak** [comment]
Ok I'm done 😄

File: src/dosbox.cpp [hunk 1]
```diff
@@ -465,7 +465,29 @@ void DOSBOX_Init()
...
+            "  ega:            IBM Enhanced Graphics Adapter (EGA).\n"
+            "  svga_paradise:  Paradise PVGA1A SVGA card (no VESA VBE; 512K vmem by default,\n"
+            "                  can be set to 256K or 1MB with 'vmemsize'). This is the\n"
+            "                  closest to IBM's original VGA adapter.\n"
+            "  svga_et3000:    Tseng Labs ET3000 SVGA card (no VESA VBE; fixed 512K vmem).\n"
+            "  svga_et4000:    Tseng Labs ET4000 SVGA card (no VESA VBE; 1MB vmem by\n"
+            "                  default, can be set to 256K or 512K with 'vmemsize').\n"
+            "  svga_s3:        S3 Trio64 (VESA VBE 2.0; 4MB vmem by default, can be set to\n"
+            "                  512K, 1MB, 2MB, or 8MB with 'vmemsize') (default)\n"
+            "  vesa_oldvbe:    Same as 'svga_s3' but with VESA VBE 1.2.\n"
```
[the remark this answers is not in the document]
Comment (johnnovak) [reply src/dosbox.cpp:488]: Done.

File: src/dosbox.cpp [hunk 2]
```diff
@@ -465,7 +465,29 @@ void DOSBOX_Init()
 	pstring->Set_values(machines);
 	pstring->SetDeprecatedWithAlternateValue("vgaonly", "svga_paradise");
 	pstring->Set_help(
-	        "The type of machine DOSBox tries to emulate ('svga_s3' by default).");
+            "Set the video adapter or machine to emulate:\n"
+            "  hercules:       Hercules Graphics Card (HGC) (see 'monochrome_palette').\n"
+            "  cga:            IBM Color Graphics Adapter (CGA). Also enables composite\n"
+            "                  video emulation (see [composite] section).\n"
+            "  cga_mono:       CGA adapter connected to a monochrome monitor (see\n"
+            "                  'monochrome_palette').\n"
```
[the remark this answers is not in the document]
Comment (johnnovak) [reply src/dosbox.cpp:473]: Good point. Changed the order of the enum values too so they're in sync with the config description.

**Comment by johnnovak** [comment]
Ready again @kcgen 😄

**Review by kcgen — APPROVED** [review]
looks good!

**Pull request merged**